test/e2e/operator_test.go

```diff
@@ -25,13 +24,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"k8s.io/client-go/util/retry"
+
 	"gopkg.in/yaml.v2"
 
 	configv1 "github.com/openshift/api/config/v1"
 	operatorv1 "github.com/openshift/api/operator/v1"
 	iov1 "github.com/openshift/api/operatoringress/v1"
 	routev1 "github.com/openshift/api/route/v1"
 
+	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
```

Comment:
Hm, I suppose grouping the `github.com/openshift/client-go` import with the `github.com/openshift/cluster-ingress-operator` imports is fine for now, but at some point, we do want to formalize and enforce import grouping rules for this repository.

Reply:
Yeah, it would be better to add this as part of the `verify`  job.